src/tournament_commands/player_commands/decklist.rs

```diff
@@ -5,5 +5,35 @@ use serenity::prelude::*;
 #[command("decklist")]
 #[description("Prints out one of your decklists.")]
 async fn decklist(ctx: &Context, msg: &Message, _: Args) -> CommandResult {
-    todo!()
+     /* Get references to needed data from context */
+     let data = ctx.data.read().await;
+     let name_and_id = data.get::<TournamentNameAndIDMapContainer>().unwrap();
+     let gld_tourns = data.get::<GuildAndTournamentIDMapContainer>().unwrap();
```

Comment:
We will need to access the tournament object, so we need to get them from the `data` map.

```suggestion
let gld_tourn_ids =  data.get::<GuildAndTournamentIDMapContainer>().unwrap();
let tourns =  data.get::<TournamentMapContainer>().unwrap();
```